pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>org.jboss</groupId>
     <artifactId>jboss-parent</artifactId>
-    <version>35</version>
+    <version>39</version>
```

Comment:
Is there a problem with `42`? `42` already contains version `3.2.3` of the surefire plugin.